README.md

```diff
@@ -59,72 +51,70 @@ Or install it yourself as:
 
     $ gem install unleash
 
-## Configure
+## Configuration
 
 It is **required** to configure:
 
-- `url` of the unleash server
-- `app_name` with the name of the runninng application.
-- `custom_http_headers` with `{'Authorization': '<API token>'}` when using Unleash v4.0.0 and later.
-
-Please substitute the example `'https://unleash.herokuapp.com/api'` for the url of your own instance.
+- `app_name` with the name of the running application
+- `url` of your Unleash server
+- `custom_http_headers` with `{'Authorization': '<YOUR_API_TOKEN>'}` when using Unleash v4+
 
 It is **highly recommended** to configure:
 
-- `instance_id` parameter with a unique identifier for the running instance.
+- `instance_id` parameter with a unique identifier for the running instance
 
 ```ruby
 Unleash.configure do |config|
   config.app_name            = 'my_ruby_app'
-  config.url                 = 'https://unleash.herokuapp.com/api'
-  config.custom_http_headers = {'Authorization': '<API token>'}
+  config.url                 = '<YOUR_UNLEASH_URL>/api'
```

Comment:
To me a placeholder makes more sense, easier to spot and replace than an example, but happy to revert